tests/test_exchange_collector.py

```diff
@@ -0,0 +1,139 @@
+import os
+from pathlib import Path
+
+import netCDF4 as nc
+import numpy as np
+import pytest
+import tomli
+from numpy.typing import NDArray
+
+from imod_coupler.logging.exchange_collector import ExchangeCollector
+
+
+def test_exchange_collector_read(tmp_path_dev: Path, output_config_toml: str) -> None:
+    """
+    Tests the happy flow of the exchange collector. It receives an input file telling it to log
+    an array called "example_flux_output", which changes through time.
+    Then it is called several times (at time 8 and time 23) and it should write the value of example_flux_output
+    at those times to netcdf
+    """
+
+    if not (os.path.isdir(tmp_path_dev)):
+        os.makedirs(tmp_path_dev)
```

Comment:
```suggestion
```

I think we can expect that the fixture works as expected or not?

Reply:
indeed,  I now removed this check